src/contrib/persistence/Akka.Persistence.Sql.Common/Snapshot/SqlSnapshotStore.cs

```diff
@@ -123,23 +123,27 @@ private async Task<object> Initialize()
             }
             catch (Exception e)
             {
-                return new Failure {Exception = e};
+                return new Status.Failure(e);
```

Comment:
Is this going to break downstream Akka.Persistence.Sql plugin implementations @Zetanova ?

Reply:
It's only internally used with `Initialize().PipeTo(Self);`
The whole akka libery didn't use the `Akka.Actor.Failure` type

Reply:
Ok, sounds good.